setup.py

```diff
@@ -32,69 +42,79 @@
 def read(*parts):
     # intentionally *not* adding an encoding option to open, See:
     #   https://github.com/pypa/virtualenv/issues/201#issuecomment-3145690
-    with codecs.open(os.path.join(HERE, *parts), 'r') as fp:
+    with codecs.open(os.path.join(HERE, *parts), "r") as fp:
         return fp.read()
 
 
 def find_version(*file_paths):
     version_file = read(*file_paths)
-    version_match = re.search(r"^__version__ = ['\"]([^'\"]*)['\"]",
-                              version_file, re.M)
+    version_match = re.search(r"^__version__ = ['\"]([^'\"]*)['\"]", version_file, re.M)
     if version_match:
         return version_match.group(1)
     else:
         raise RuntimeError("Unable to find version string.")
 
 
-WEBSITE = 'https://alan-turing-institute.github.io/sktime/'
-DISTNAME = 'sktime'
-DESCRIPTION = 'scikit-learn compatible Python toolbox for machine learning ' \
-              'with time series'
-with codecs.open('README.rst', encoding='utf-8-sig') as f:
+WEBSITE = "https://alan-turing-institute.github.io/sktime/"
+DISTNAME = "sktime"
+DESCRIPTION = (
+    "scikit-learn compatible Python toolbox for machine learning " "with time series"
```

Comment:
Another quick update here since we're on it:
```suggestion
    "A unified Python toolbox for machine learning with time series"
```